packages/terra-functional-testing/src/reporters/spec-reporter/wdio-spec-reporter.js

```diff
@@ -120,7 +120,12 @@ class SpecReporter extends WDIOReporter {
     if (spec.indexOf(`${path.sep}packages${path.sep}`) > -1) {
       return spec.split(`${path.sep}packages${path.sep}`).pop().split(path.sep)[0];
     }
-
+    const packageJsonPath = path.join(process.cwd(), 'package.json');
+    if (fs.existsSync(filepath)) {
```

Comment:
oops, you'll have to update the path variable because my suggestion did not propagate beyond line 123

Reply:
Fixed here https://github.com/cerner/terra-toolkit/pull/613/commits/c8698e39c24fef876e728ae9139db62ed611116a